tests/lent/tbasic_lent_check.nim

```diff
@@ -14,3 +15,31 @@ proc main =
   doAssert(not compiles(passToVar(viewInto(x))))
 
 main()
+
```

Comment:
also un-disable the disabled test case introduced in https://github.com/nim-lang/Nim/pull/15960

Reply:
I am not fixing VM or JS in this PR

Reply:
this one: see https://github.com/nim-lang/Nim/pull/15960/files
```nim
  block:
    when not defined(cpp): # pending bug #15958
      let a = [11,12]
      doAssert byLent(a) == [11,12]
    let a2 = (11,)
    doAssert byLent(a2) == (11,)
```

Reply:
=> fixing this in https://github.com/nim-lang/Nim/pull/16075